styles.css

```diff
@@ -274,6 +563,90 @@ width и height для блока, а не его содержимого, */
   .main__wrap {
     width: 90%;
   }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5654195)